.gitignore

```diff
@@ -127,9 +127,15 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
-tests/local_data
-tests/data
+tests/scripts/data
+tests/data/
 pv-solar-farm-forecasting.code-workspace
 .vscode
 .gitattributes
+.flake8
+.pre-commit-config.yaml
+environment.yml
+pydoc-markdown.yml
+requirements.txt
```

Comment:
You shouldn't ignore this and the environment.yml, or pre commit config, as they are used for building and linting

Reply:
yes, I updated this!